2021/2021-01/Submissions/AlbertHenry/game.py

```diff
@@ -1 +1,36 @@
-""" Test submissions """
+from random import choice
+
+opts = dict(R = "Rock", P = "Paper", S = "Scissors")
+
+win_against = dict(R = "S", P = "R", S = "P")
+
+def who_win(opt1, opt2):
+    if opt1 == opt2:
+        result = None
+    elif win_against[opt1] == opt2:
+        result = opt1
+    else:
+        result = opt2
+    return result
+
+player = False
+cpu = choice(list(opts.keys()))
+
+while player == False:
```

Comment:
Nice running loop forever could be done with 
```python
while True:
```

So you can remove the player and cpu choice before running